From 11c3c1203cefda72a740d37cb9d6980875ff4f6a Mon Sep 17 00:00:00 2001
From: John Brooks <john.brooks@jollamobile.com>
Date: Thu, 19 Jun 2014 04:53:07 -0600
Subject: [PATCH 2/4] Don't depend on Qt5Widgets for build

This is used only for examples, which we don't build.
---
 cmake/modules/FindQt5.cmake | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/cmake/modules/FindQt5.cmake b/cmake/modules/FindQt5.cmake
index b9a35e16..07d5907a 100644
--- a/cmake/modules/FindQt5.cmake
+++ b/cmake/modules/FindQt5.cmake
@@ -66,22 +66,20 @@ find_package(Qt5Core ${REQUIRED_QT_VERSION} REQUIRED)
 find_package(Qt5DBus ${REQUIRED_QT_VERSION} REQUIRED)
 find_package(Qt5Gui ${REQUIRED_QT_VERSION} REQUIRED)
 find_package(Qt5Test ${REQUIRED_QT_VERSION} REQUIRED)
-find_package(Qt5Widgets ${REQUIRED_QT_VERSION} REQUIRED)
 find_package(Qt5Network ${REQUIRED_QT_VERSION} REQUIRED)
 find_package(Qt5Xml ${REQUIRED_QT_VERSION} REQUIRED)
 
 # Copy includes and library names into the same style as pkgconfig used for Qt4
-set(QT_INCLUDES ${Qt5Core_INCLUDE_DIRS} ${Qt5DBus_INCLUDE_DIRS} ${Qt5Gui_INCLUDE_DIRS} ${Qt5Network_INCLUDE_DIRS} ${Qt5Test_INCLUDE_DIRS} ${Qt5Widgets_INCLUDE_DIRS})
+set(QT_INCLUDES ${Qt5Core_INCLUDE_DIRS} ${Qt5DBus_INCLUDE_DIRS} ${Qt5Gui_INCLUDE_DIRS} ${Qt5Network_INCLUDE_DIRS} ${Qt5Test_INCLUDE_DIRS})
 
 set(QT_QTCORE_LIBRARY ${Qt5Core_LIBRARIES})
 set(QT_QTDBUS_LIBRARY ${Qt5DBus_LIBRARIES})
 set(QT_QTGUI_LIBRARY ${Qt5Gui_LIBRARIES})
 set(QT_QTNETWORK_LIBRARY ${Qt5Network_LIBRARIES})
 set(QT_QTTEST_LIBRARY ${Qt5Test_LIBRARIES})
-set(QT_QTWIDGETS_LIBRARY ${Qt5Widgets_LIBRARIES})
 set(QT_QTXML_LIBRARY ${Qt5Xml_LIBRARIES})
 
-set(QT_LIBRARIES ${QT_QTCORE_LIBRARY} ${QT_QTDBUS_LIBRARY} ${QT_QTGUI_LIBRARY} ${QT_QTNETWORK_LIBRARY} ${QT_QTTEST_LIBRARY} ${QT_QTWIDGETS_LIBRARY} ${QT_QTXML_LIBRARY})
+set(QT_LIBRARIES ${QT_QTCORE_LIBRARY} ${QT_QTDBUS_LIBRARY} ${QT_QTGUI_LIBRARY} ${QT_QTNETWORK_LIBRARY} ${QT_QTTEST_LIBRARY} ${QT_QTXML_LIBRARY})
 
 SET(QT_VERSION_MAJOR ${Qt5Core_VERSION_MAJOR})
 SET(QT_VERSION_MINOR ${Qt5Core_VERSION_MINOR})
-- 
2.20.1

